src/browser.js

```diff
@@ -39,7 +39,7 @@ var NOT_SUPPORTED = 'The browser target is not supported: %target%';
 module.exports = function (opts) {
 
     opts = opts || {};
-    var target = opts.target || 'chrome';
+    var target = opts.target || 'default';
```

Comment:
Hey @TimBarham!

Thanks for chiming in. I didn't know that about visual studio 2017. I can definitely bump it up to 2.